helpdesk/signals.py

```diff
@@ -0,0 +1,7 @@
+import django.dispatch
+
+# create a signal for *TicketForm
```

Comment:
small typo here I think

Reply:
It was intended as TicketForm and PublicTicketForm are emitting the signal

Reply:
Oh ok, I understand now. Yeah it was strange for a typo